### PR TITLE
Fixes system_analysis bug #132 when pv_nameplate is passed into syste…

### PR DESCRIPTION
**Pull request by cdeline**
…m_analysis object.
pretty straightforward- sets pv_nameplate = self.pv_nameplate if the value exists.

Also, unrelated edit adding optional input to plotting.degradation_summary_plots(hist_nbins=200). Custom definition of the number of bins in the Rd histogram.  For data sets with small numbers of YoY pairs, the default is to only plot 1/40th of the datapoints.  This sets the default number of bins to 200, but allows it to be manually set.

### CODE REVIEW

**Comment by cdeline** [comment]
I checked this out, and tests are passing.  Example notebooks are only slightly changed - the default histogram plot uses 75 bins which is close to the value used in our example notebook.

File: rdtools/plotting.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 
 def degradation_summary_plots(yoy_rd, yoy_ci, yoy_info, normalized_yield,
-                              hist_xmin=None, hist_xmax=None, scatter_ymin=None,
-                              scatter_ymax=None, plot_color=None, summary_title=None,
-                              scatter_alpha=0.5):
+                              hist_xmin=None, hist_xmax=None,  bins=75, 
+                              scatter_ymin=None, scatter_ymax=None, plot_color=None, 
+                              summary_title=None, scatter_alpha=0.5):
```
Comment (mdeceglie) [inline rdtools/plotting.py]: I'm generally in favor with adding this control of the bin count. This does also change the default behavior from `yoy_values//40` to 75. I want to make sure everyone agrees. @dirkjordan ? 

If not we can make the default `None` with a conditional that sets bins to `yoy_values//40`

**Comment by mdeceglie** [comment]
Let's revert the plotting changes, they are handled in #138 , and merge just the bug fix

**Comment by kanderso-nrel** [comment]
@mdeceglie @cdeline PTAL

**Comment by cdeline** [comment]
OK, so this removes all of the plotting changes and leaves the one-line bug fix to set `pv_nameplate = self.pv_nameplate` which is a pretty trivial change.  I'd review and approve this myself, but I can't.  :)

**Pull request merged**